libr/main/radare2.c

```diff
@@ -1328,15 +1328,21 @@ R_API int r_main_radare2(int argc, char **argv) {
 		r_flag_space_set (r.flags, NULL);
 		/* load <file>.r2 */
 		{
-			char f[128];
-			snprintf (f, sizeof (f), "%s.r2", pfile);
+			char* f = r_str_newf ("%s.r2", pfile);
+			const int f_len = strlen (f);
+			const char *prefix = "dbg://";
```

Comment:
i think it will be better to use : 

```
const char *peekaboo = strstr (pfile, "://");
char *path = peekaboo? peekaboo +3: pfile;
if (r_file_exisits(path)) { ...
```

Reply:
so we handle all the possible prefixes (may be useful for iihex:// mmap:// and others despite have some undesired behaviours on others, but as long asi ts asking before executing it should be fine